<launch>
  <!-- <arg name="controllers" default="joint_state_controller hitbot_joint_traj_controller" />
  <rosparam file="$(find hitbot_hw)/config/controllers.yaml" command="load"/>
  <node name="ros_control_controller_spawner" pkg="controller_manager" type="spawner" 
      respawn="false" output="screen" args="$(arg controllers)"/> -->

  <!-- joint_state_controller -->
  <rosparam file="$(find hitbot_hw)/config/joint_state_controller.yaml" command="load"/>
  <node name="joint_state_controller_spawner" pkg="controller_manager" type="controller_manager" 
        args="spawn joint_state_controller" respawn="false" output="screen"/>

  <!-- <node type="hitbot_hw_node" pkg="hitbot_hw" name="hitbot_hw_node" output="screen">
  </node> -->

  <!-- start this controller -->
  <rosparam file="$(find hitbot_hw)/config/arm_controllers.yaml" command="load"/>
  <node name="arm_controller_spawner" pkg="controller_manager" type="controller_manager" 
        args="spawn robot_arm_controller" respawn="false" output="screen"/>

  <!-- load other controllers -->
  <node name="ros_control_controller_manager" pkg="controller_manager" type="controller_manager" 
        respawn="false" output="screen" args="load hitbot_joint_group_position_controller" />


  <node type="hitbot_hw_node" pkg="hitbot_hw" name="hitbot_hw_node" output="screen">
  </node>

</launch>